### DISCUSSION TITLE
specification/markdown/chinese_copywriting_guidelines/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#57] [Announcements]
# specification/markdown/chinese_copywriting_guidelines/

https://tooldiy.ry.rs/specification/markdown/chinese_copywriting_guidelines/

**Comment by Mc-GrowlR** [comment]
如果使用 `Obsidian` ，可以添加插件来帮助使用者自动进行文档排版，比如文中所述的中英文之间的空格问题，这方面 Ob 的插件已经做的很完善了。
